content/automate/ManageIQ/Transformation/Infrastructure/VM/vmwarews.class/__methods__/utils.rb

```diff
@@ -7,53 +7,44 @@ module VMware
             class Utils
               require 'rbvmomi'
 
-              def initialize(handle = $evm)
-                @handle = handle
-              end
-
-              def main
+              def self.host_fingerprint(host)
```

Comment:
@fdupont-redhat do you want to pass a handle into these class methods so that they can be tested

Reply:
@mkanoor do I need a handle to test them even though I don't use it in the method ?

Reply:
@fdupont-redhat A missing handle means we are not logging any errors in the method we are expecting the exception to report an error. If you plan on writing specs it might be better to have a handle.

Reply:
@mkanoor 
I've added the handle but now Rubocop complains that I don't use it.
I probably miss something in the implementation of specs, as I don't see what this will bring. If my method raise, it will be caught by something like `expect(described_class.provider_connection).to raise(errormsg)`.